INSTALL.md

```diff
@@ -14,7 +14,7 @@ On Debian-based systems:
 ```
 $ sudo apt-get update && \
   sudo apt-get install -y build-essential \
-  libssl-dev uuid-dev libseccomp-dev \
+  libssl-dev uuid-dev libgpgme11-dev libseccomp-dev \
```

Comment:
Is this an absolute need now, or did it get dropped somewhere on master?

Reply:
The `gpgme` requirement was removed from master in #4354 .  The `release-3.4` branch probably doesn't have it.

But, it would be good to *not* add that requirement back in.

Reply:
that's what I was trying to figure out.

According to me :man_shrugging: it's not required anymore:

```
commit 29ff86a76fad5f620566c32689ec7dce0a51c74c
Author: Marcelo E. Magallon <marcelo@sylabs.io>
Date:   Fri Aug 30 11:04:11 2019 -0600

    Remove gpgme as a dependency for building singularity

    github.com/containers/image can switch between using gpgme and openpgp
    for processing PGP signatures. It does this using a build tag.

    We are _always_ passing the build tag to have it use openpgp.

    Remove gpgme as a dependency for building singularity as it is not
    required.

    Signed-off-by: Marcelo E. Magallon <marcelo@sylabs.io>
```

I'll fix this.